Source/cairo/CairoFontInfo.m

```diff
@@ -71,16 +82,48 @@ - (BOOL) setupAttributes
       return NO;
     }
 
-  // We must not leave the hinting settings as their defaults,
```

Comment:
How do you make sure that the problem mentioned in the comment won't affect the text drawing?

Reply:
I don't. In fact, I make note that it does. For some reason, __disabling__ the hinting of metrics causes menus to get cut off by one pixel on my setup -- which is actually the opposite of what should happen. If something is going to screw up, it should happen when you enable hinting of metrics, not when you turn it off.